rootfs/standard/var/pynode/application_info.py

```diff
@@ -738,6 +754,39 @@ def restart_application(short_name):
     except Exception as e:
         return False
 
+def backup_data_folder(app_data):
+    log_message("  Running backup_data_folder...")
+
+def restore_data_folder(app_data):
+    log_message("  Running restore_data_folder...")
```

Comment:
It looks like backup and restore are still unimplemented.

Reply:
Yes. I thought that would be nice to implement reset first and then backup and restore later on. 
I have both code on phoenixd App which is working, but I need to adjust those to match this 1st part (reset).

Or, would you like to see all three finctions implemented on one PR and released same time?